### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by lydell**
This PR is a cherry-pick of #442. The idea is let the little fixes from that PR be merged into master earlier and to reduce the diff in #442 to make that PR easier to review. This will cause some merge conflicts in #442 but I think it’s worth it.

### CODE REVIEW

**Comment by harrysarson** [comment]
Looks good! Have you had a chance to look at https://github.com/lydell/node-test-runner/pull/1? I guess if we land this I can then rebase that PR and point it against master branch of this repository.

**Comment by lydell** [comment]
Thanks for linking to https://github.com/lydell/node-test-runner/pull/1! It looks like the default Watch setting for forks is “Not watching”, so I didn’t receive notifications for it. I’ll have a look at this tomorrow.

**Comment by lydell** [comment]
I ended up doing this straight away after all. I cherry-picked two of your commits and implemented the others in other ways.

File: .eslintrc.json [hunk 1]
```diff
@@ -13,9 +13,23 @@
...
                 "mocha": true
+            },
+            "rules": {
+                "mocha/handle-done-callback": "error",
+                "mocha/no-exclusive-tests": "error",
+                "mocha/no-exports": "error",
+                "mocha/no-identical-title": "error",
+                "mocha/no-nested-tests": "error",
+                "mocha/no-setup-in-describe": "error",
+                "mocha/no-skipped-tests": "error"
```
Comment (harrysarson) [inline .eslintrc.json:32]: Fantastic!

File: tests/flags.js [hunk 2]
```diff
@@ -265,7 +262,7 @@ describe('flags', () => {
     }).timeout(60000);
   });
 
-  describe.only('--compiler', () => {
+  describe('--compiler', () => {
```
Comment (harrysarson) [inline tests/flags.js:265]: Sorry!

**Review by harrysarson — APPROVED** [review]
Love it.

**Comment by harrysarson** [comment]
Do you want me to merge? Or are there more commits lined up?

**Comment by lydell** [comment]
You can merge! :+1:

**Pull request merged**